Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -86,28 +91,25 @@ def toLinHom : BilinForm R M →ₗ[R] M →ₗ[R] M →ₗ[R] R where
     intro y
     simp only [toLinHomAux₂, toLinHomAux₁, LinearMap.coe_mk, LinearMap.smul_apply, smul_apply,
       AddHom.coe_mk]
-#align bilin_form.to_lin_hom BilinForm.toLinHom
+#align bilin_form.to_lin_hom LinearMap.BilinForm.toLinHom
 
-@[simp]
-theorem toLin'_apply (A : BilinForm R M) (x : M) : ⇑(toLinHom A x) = A x :=
+theorem toLin'_apply (A : BilinForm R M) (x : M) : toLinHom.toFun A x = A x :=
```

Comment:
The `toFun` shouldn't be needed here.

Reply:
```
theorem toLin'_apply (A : BilinForm R M) (x : M) : toLinHom A x = A x :=
  rfl
```
gives me:
```
function expected at
  toLinHom
term has type
  BilinForm ?m.30887 ?m.30888 →ₗ[?m.30887] ?m.30888 →ₗ[?m.30887] ?m.30888 →ₗ[?m.30887] ?m.30887
```

Reply:
Similarly for `⇑(toLinHom A x) = A x`

Reply:
Try adding an explicit type annotation to `toLinHom`

Reply:
Thanks, I've replaced the other additions of `toFun` with explicit type annotations as well.